File: Python Word Examples/DigitalSignatureApiDemo/addDigitalSignatureInRequest.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.digital_signature_api.DigitalSignatureApi(configuration)

file_name = 'Template.docx'
pfx_file = 'D:/input/Gary.pfx'
secure_password = 'e-iceblue'
extra = {'folder': 'input',
         'dest_file_path': 'output/AddDigitalSignatureInRequest.docx'}
result = api.add_digital_signature_in_request(file_name, pfx_file, secure_password, **extra)

File: PHP Word Examples/DigitalSignatureApiDemo/addDigitalSignatureInRequest.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\ConvertApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new DigitalSignatureApi($configuration);

$input_file ="D:/data/Template.docx";
$pfx_file = "D:/data/Demo.pfx";
$secure_password = 'e-iceblue';
$doc_password = null;
$result = $apiInstance->addDigitalSignatureInRequest(
    $input_file,
    $pfx_file,
    $secure_password,
    $doc_password
);